pkg/controller/virtualmachinevolumeexport/pvc.go

```diff
@@ -0,0 +1,109 @@
+package virtualmachinevolumeexport
+
+import (
+	"context"
+	goerrors "errors"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog"
+	hc "kubevirt-image-service/pkg/apis/hypercloud/v1alpha1"
+	vmv "kubevirt-image-service/pkg/controller/virtualmachinevolume"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+func (r *ReconcileVirtualMachineVolumeExport) syncPvc() error {
```

Comment:
Please give a more specific name than just `syncPvc`. What does PVC mean?

Reply:
PVC indicates the volume containing the image file converted from raw format of vmv to be exported to qcow2. How about `syncExportPvc`?